.github/workflows/e2e.yml

```diff
@@ -638,6 +638,57 @@ jobs:
           vertica-deployment-method: vclusterops
           communal-storage-type: s3
   
+  upload-operator-to-private-repo:
+    needs: [
+      build,
+      e2e-leg-1-vcluster,
+      e2e-leg-2-vcluster,
+      e2e-leg-3-vclusterops-previous-release,
+      e2e-leg-3-vcluster,
+      e2e-leg-4-vcluster,
+      e2e-leg-5-vcluster,
+      e2e-leg-6-vcluster,
+      e2e-leg-6-vcluster-revivedb,
+      e2e-leg-7-vcluster,
+      e2e-leg-8-vcluster-offline,
+      e2e-leg-8-vcluster-online,
+      e2e-leg-9-vcluster,
+      e2e-leg-10-vcluster,
+      e2e-server-upgrade-vcluster,
+      e2e-operator-upgrade,
+      e2e-udx-vcluster
+    ]
+    runs-on: ubuntu-latest 
+    if: ${{ always() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && contains('server daily version', github.event.workflow_run.display_title) && contains('verticareleng', github.actor) }}  
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        path: vertica-kubernetes
+        fetch-depth: 0
+
+    - name: Save current version to env VERSION
```

Comment:
We don't need to infer the version again(remove this step). You can either get the version from "build.outputs.operator-image"(I perfer this way) or "github.sha".

Reply:
The value of "build.outputs.operator-image": is something like: ghcr.io/vertica/verticadb-operator:93d02b6d9356416a54317e291a0638163b84b181, which contains both repo and tag, not the version value. And what we are looking for here is the operator version, something like: VERSION=24.4.0-0. And we need both version and version-sha

Reply:
Per discussion, used the operator-image output, changed updated in https://github.com/vertica/vertica-kubernetes/pull/929/commits/53074ab1b34e498ad3e6c0e6b2ac2f2e4d8c319a